test/abis.ts

```diff
@@ -0,0 +1,222 @@
+export const tokenAbi = [
```

Comment:
Is this the ABI for an ERC20? If so, could you modify the filename to reflect that?